openseadragon-scalebar.js

```diff
@@ -460,14 +475,14 @@
         return ratio * viewportZoom;
     }
 
-    function getScalebarSizeAndText(ppm, minSize, unitSuffix, handlePlural) {
+    function getScalebarSizeAndText(ppm, minSize, unitSuffix, handlePlural, spacer = ' ') {
```

Comment:
I would rather not use this syntax for compatibility with older browsers.
Could you remove the `= ' '` here and add `spacer = spacer === undefined ? ' ' : spacer;` on the next line?